_posts/2019-02-08-react-checkers.html

```diff
@@ -1,11 +1,47 @@
 ---
 title: Checkers
 thumbnail: -SITE_URL-/images/react-checkers.jpg
-excerpt: "(Coming soon) A checkers game made using React."
+excerpt: "A checkers game made using React and Redux."
 # githublink: "#"
 bitbucketlink: "https://bitbucket.org/Dries_Sercu/react-checkers"
 demolink: "https://sdries-react-checkers.herokuapp.com/"
 ---
 <p>
-	I'm working on the article as we speak. Feel free to already check out the code and the game.
+	When I decided to learn react, like most people I ended up following along with the <a href="https://reactjs.org/tutorial/tutorial.html" target="_blank">tutorial on their website</a>. The result is also publicly available <a href="https://bitbucket.org/Dries_Sercu/react-tutorial-tic-tac-toe" target="_blank">here</a>, however rather dull.
+</p>
+<p>
+	Because I wanted to have some fun when learning the react basics I decided to make a checkers game.
+	This has been a very educational project when it came to react and even redux (which was new to me).
```

Comment:
This has been a very educational project when it came to React and even Redux (which was new to me).